amethyst_ui/src/pass.rs

```diff
@@ -216,13 +194,12 @@ impl Pass for DrawUi {
         // change.
         self.cached_draw_order
             .cache
-            .sort_unstable_by(|&(z1, _), &(z2, _)| z2.partial_cmp(&z1).unwrap_or(Ordering::Equal));
+            .sort_unstable_by(|&(z1, _), &(z2, _)| z1.partial_cmp(&z2).unwrap_or(Ordering::Equal));
```

Comment:
If we're changing this then there's a lot of UI documentation that also needs to change.  Please look over the docs for `UiTransform` and try a few greps just to make sure we don't leave any incorrect info in our docs.